hw/top_earlgrey/data/clocks.xdc

```diff
@@ -21,6 +21,12 @@ create_generated_clock -name clk_io_div2 -source [get_pin ${u_pll}/CLKOUT0] -div
 set u_div4 top_*/u_clkmgr_aon/u_no_scan_io_div4_div
 create_generated_clock -name clk_io_div4 -source [get_pin ${u_pll}/CLKOUT0] -divide_by 4 [get_pin ${u_div4}/u_clk_div_buf/gen_xilinx.u_impl_xilinx/gen_fpga_buf.bufg_i/O]
 
+# the step-down mux is implemented with a LUT right now and the mux switches on the falling edge. 
+# therefore, Vivado propagates both clock edges down the clock network.
+# this implementation is not ideal - but we can at least tell Vivado to only honour the rising edge for 
+# timing analysis.
+set_clock_sense -positive top_earlgrey/u_clkmgr_aon/u_no_scan_io_div2_div/gen_div2.u_div2/gen_fpga_buf.bufg_i_i_1__0/O
```

Comment:
Would you mind to replace "top_earlgrey" with "top_*" to ensure the constraint can also be applied to other top levels? Currently, we use one `clocks.xdc` that is symlinked.

Reply:
Sure - done!